src/basins/compute_ba.jl

```diff
@@ -123,10 +124,19 @@ Compute an estimate of the basins of attraction of a higher-dimensional dynamica
 on a projection of the system dynamics on a two-dimensional plane.
 
 Like [`basins_map2D`](@ref), `xg, yg` are ranges defining the grid of initial conditions
-on the plane.
-
-Refer to [`basins_map2D`](@ref) for detailed information on the
-computation and the structure of `basin` and `attractors`.
+on the plane. Refer to [`basins_map2D`](@ref) for more details.
+
+This function can be used to make attractor basins of higher dimension. E.g. to make
+a 3D basin you can make many 2D basins and concatenate them. For example:
```

Comment:
```suggestion
3D basins you can make many 2D basins and concatenate them. For example:
```

Not sure when to use singular.